client/placeholder/README.md

```diff
@@ -0,0 +1,13 @@
+# Placeholder
+
+*Note*: This placeholder is not used in the deployment self-contained in this repo, but in another project. 
```

Comment:
suggestion: link to that project?
```suggestion
*Note*: This placeholder is not used in the deployment self-contained in this repo, but in [another project](https://github.com/GoogleCloudPlatform/terraform-dynamic-python-webapp). 
```

Reply:
Not going to accept this change, given the pending repo move